packages/bsb-core/bsb/config/_attrs.py

```diff
@@ -1025,8 +1135,10 @@ def __ref__(self, instance, root):
         return self.resolve_reference_list(instance, remote, remote_keys)
 
     def resolve_reference_list(self, instance, remote, remote_keys):
```

Comment:
The arguments to `resolve_reference_list` are a bit weird now ... follow along:

* We create a `cfgreflist`, together with the remote keys `cfgreflist._ref_list`.
* We set it on the instance.
* We pass the instance, remote, and remote_keys (= `cfgreflist._ref_list`) to the function
* We retrieve the `cfgreflist` from the instance...

But ... we had `cfgreflist` in our hands already, and it has the remote keys on it.

Check which callers of `resolve_reference_list` there are. I think that probably now, all the information is already available on the instance and things can be simplified drastically.